test/test_grade.py

```diff
@@ -63,6 +62,8 @@ def generate_autograder_zip(pdfs=False):
             config = TEST_FILES_PATH + "otter_config.json" if pdfs else None,
             no_environment = True,
         )
+        with zipfile.ZipFile(TEST_FILES_PATH + "autograder.zip", "a") as zip_ref:
+            utils.zip_folder(zip_ref, os.path.dirname(os.path.dirname(os.path.abspath(__file__))), exclude=[".git", "logo", "test"])
```

Comment:
I think it might be better to replace `os.path.dirname(os.path.dirname(os.path.abspath(__file__))` with `os.getcwd()` since tests are run from the root of the repo.